js/utils.js

```diff
@@ -415,6 +415,162 @@ Fliplet.Registry.set('dynamicListUtils', (function() {
     }
   }
 
+  function getDateModifiedValues(options) {
+    var isEntryContainsTime = /^([0-1][0-9]|[2][0-3]):([0-5][0-9])$/.test(options.date);
+    var deviceTimeZone = moment.tz.guess();
+    var getDate = options.isDeviceTimeZoneEnabled ? moment.tz.setDefault(deviceTimeZone) : moment.utc;
+
+    switch (options.filterModifier) {
+      case 'today':
+        return {
+          today: moment().startOf('day'),
+          inputDate: isEntryContainsTime
+            ? moment(options.date, 'HH.mm').startOf('day')
+            : moment(options.date).startOf('day')
+        };
+
+      case 'now':
+        return {
+          today: getDate().startOf('minute'),
+          inputDate: isEntryContainsTime
```

Comment:
I don't see any checks to verify whether a value that's used for time comparison contains actual time.

![image](https://user-images.githubusercontent.com/290733/108637287-faaa4c80-7481-11eb-944c-8cf53ecb0b5d.png)

Reply:
Fixed